### PR TITLE
De-densify Bazel vision page

### PR DESCRIPTION
**Pull request by gregestren**
Old site: https://docs.bazel.build/versions/2.1.0/bazel-vision.html

New (with new styling):

Screen Shot 2020-02-18 at 10 39 32 PM
Screen Shot 2020-02-18 at 10 39 48 PM
Screen Shot 2020-02-18 at 10 39 55 PM

Serves #10794.

### CODE REVIEW

File: site/docs/bazel-vision.md [hunk 1]
```diff
@@ -5,40 +5,49 @@ title: Bazel Vision
...
+inputs doesn't introduce the risk of corruption) and distributability (build
+actions are isolated and can be offloaded). By minimizing the work needed to do
+a correct build and parallelizing that work across multiple cores and remote
+systems, Bazel can make any build fast.
+
+Bazel's abstraction layer — instructions specific to languages, platforms, and
+toolchains implemented in a simple extensibility language — allows it to be
+easily applied to any context.
 
 ## Bazel Core Competencies
```
Comment (gregestren) [inline site/docs/bazel-vision.md]: For this and all other level2+ headers, should we use **Capitalize All Words** formatting or **Only first word**?

This and the the very next header disagree. And we have inconsistent use all throughout the docset. I'm happy to do a docset-wide refactoring if we can agree on the answer.

I suspect the answer is **Only first word**. But I have no authority to point to.

**Comment by aiuto** [comment]
We must have a style guide for Google doc writers. We should adopt its
recommendation and not have a debate about it.

File: site/docs/bazel-vision.md [hunk 2]
```diff
@@ -5,40 +5,49 @@ title: Bazel Vision
...
     committed. This remains true even as repos grow, projects span multiple
     repos, and multiple languages are introduced. Infrastructure does not force
     developers to trade test coverage for build speed.
 
-**We believe Bazel has the potential to fulfill this vision.** Bazel was built
+---
+
+**We believe Bazel has the potential to fulfill this vision.**
+
+Bazel was built
```
Comment (aiuto) [inline site/docs/bazel-vision.md]: will md-format rewrite this?

**Comment by gregestren** [comment]
> We must have a style guide for Google doc writers. We should adopt its recommendation and not have a debate about it.


https://developers.google.com/style/capitalization says to use sentence case for *everything*, including page titles.

So... I'll go with that (in a followup PR).

File: site/docs/bazel-vision.md (hunk 2 above)
Comment (gregestren) [reply site/docs/bazel-vision.md]: Fixed.

**Pull request closed without merging**